docs/docs/apis/overview.md

```diff
@@ -0,0 +1,80 @@
+# APIs
+
+## Overview
+
+Nitric APIs make it dead simple to define and map simple functions to HTTP and REST APIs.
```

Comment:
Do you need both HTTP and REST - seems a bit redundant?

Reply:
Removed the REST reference. I kept the HTTP, since that's the umbrella term